RocketChat/Rocket.Chat.ReactNative
# [NEW] Toggle analytics events

## Proposed changes
Add switch to toggle analytics events.

## How to test or reproduce
- In analytics Debug View, try logging some events in the app.
- Then go to Settings and turn Log analytics events off.
- After that try logging the events again, it shouldn't appear any event in the Debug View.

## Screenshots
![ezgif com-gif-maker](https://user-images.githubusercontent.com/29265857/92501337-c6451b80-f1d4-11ea-8ab0-42f8d07d8026.gif)

## Types of changes




## Checklist



## Further comments
